browser/ui/webui/brave_welcome_ui.cc

```diff
@@ -157,6 +161,10 @@ void WelcomeDOMHandler::RegisterMessages() {
       "setMetricsReportingEnabled",
       base::BindRepeating(&WelcomeDOMHandler::HandleSetMetricsReportingEnabled,
                           base::Unretained(this)));
+  web_ui()->RegisterMessageCallback(
+      "setBraveThemeType",
+      base::BindRepeating(&WelcomeDOMHandler::SetBraveThemeType,
+                          base::Unretained(this)));
```

Comment:
I don't think you need to add a new message type since this page already has access to [the `chrome.braveTheme.setBraveThemeType('Light' | 'Dark' | '')` API](https://github.com/brave/brave-core/blob/9a2f3e4594c89c9bb49a10bb81d465433f0a3687/common/extensions/api/brave_theme.json#L65-L76)

Reply:
done